zk_toolbox/crates/zk_inception/src/commands/prover/args/init.rs

```diff
@@ -440,4 +473,62 @@ impl ProverInitArgs {
 
         cloud_type.into()
     }
+
+    fn ask_copy_configs(&self) -> bool {
+        self.copy_configs
+            .unwrap_or_else(|| PromptConfirm::new(MSG_COPY_CONFIGS_PROMPT).ask())
+    }
+
+    fn fill_database_values_with_prompt(
+        &self,
+        config: &ChainConfig,
+    ) -> Option<ProverDatabaseConfig> {
+        let setup_database = self
+            .setup_database
+            .clone()
+            .unwrap_or_else(|| PromptConfirm::new("Do you want to setup the database?").ask());
+
+        if setup_database {
+            let DBNames { prover_name, .. } = generate_db_names(config);
+            let chain_name = config.name.clone();
+
+            let dont_drop = self.dont_drop.clone().unwrap_or_else(|| {
+                !PromptConfirm::new("Do you want to drop the database?")
+                    .default(true)
+                    .ask()
+            });
+
+            if self.use_default.clone().unwrap_or_else(|| {
+                PromptConfirm::new(MSG_USE_DEFAULT_DATABASES_HELP)
+                    .default(true)
+                    .ask()
+            }) {
+                Some(ProverDatabaseConfig {
+                    database_config: DatabaseConfig::new(DATABASE_PROVER_URL.clone(), prover_name),
+                    dont_drop,
+                })
```

Comment:
I'd rather use promts with db name and so on. 
Use default needs, when you want to specify it from the console